lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -217,12 +238,17 @@ def create(self, subsection, block_structure=None, read_only=False):
 
         If read_only is True, doesn't save any updates to the grades.
         """
-        block_structure = self._get_block_structure(block_structure)
+        self._log_event(
+            log.warning, u"create, read_only: {0}, subsection: {1}".format(read_only, subsection.location)
```

Comment:
Here on the other hand it's useful because we want to serialize `subsection.location` to unicode (I think).